kolibri/logger/management/commands/userdata_utils.py

```diff
@@ -0,0 +1,327 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import datetime
+import random
+
+from django.db.models import Max, Min, Sum
+from django.db.models.query import F
+from kolibri.auth.constants.role_kinds import ADMIN
+from kolibri.auth.filters import HierarchyRelationsFilter
+from kolibri.auth.models import Classroom, Facility, FacilityUser
+from kolibri.content.models import ContentNode
+from kolibri.core.device.models import DeviceSettings, DevicePermissions
+from kolibri.logger.models import AttemptLog, ContentSessionLog, ContentSummaryLog, MasteryLog
+from le_utils.constants import content_kinds
+
+
+def provision_device():
```

Comment:
Can consolidate this with this: https://github.com/learningequality/kolibri/blob/release-v0.6.x/kolibri/auth/test/helpers.py#L17